### ISSUE TITLE
Improve documentation of optimizers: include LaTeX formulas

### ISSUE THREAD

**Issue by lorenzoritter** [fchollet/keras#2911]
Hi all,

I have extensively used Keras in the last few months and it has proven a great program. For many problems, the documentation helps a lot. However, it includes[ very little information about the different optimizers](http://keras.io/optimizers/). In particular, the actual optimization functions are not included in the documentation.

It is hard to set good parameters for the optimizers without knowing the optimization functions. Of course, you can look for the functions elsewhere online, but it would be easier for user to have the functions right there in the documentation.

As far as I know, it is not completely trivial to include LaTeX functions in the Markdown documentation files. Else, I would have already done so and created a pull request. I possible solution I found is using [Python-Markdown-Math](https://github.com/mitya57/python-markdown-math) as a Markdown documentation. I tried it out and it works. I can now add LaTeX functions to the documentation. The downside is that contributors have to install an extra package in order to contribute to the documentation.

What do you think? Would it be an advantage to include the formulas of the optimizers? Is there an easier solution implement them?
